health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/components/UploadDataMapping.js

```
@@ -0,0 +1,701 @@
+                  currentCategories: currentCategories,
+                  payload: {
+                    row: showPopUp,
+                    selectedBoundary: selectedBoundary,
+                    selectedLevel: selectedLevel,
+                  },
+                });
+                setShowPopUp(false);
+                setSelectedLevel(null);
+                setSelectedBoundary(null);
+              }}
+            />,
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)